README.md

```diff
@@ -1535,7 +1535,7 @@ But Atrium shows where it goes wrong and even gives a possible hint:
 expect: 9.99        (kotlin.Float <1234789>)
 ◆ to be (error ± 0.01): 10.0        (kotlin.Float <1234789>)
     » failure might be due to using kotlin.Float, see exact check on the next line
-    » exact check is |9.989999771118164 - 10.0| = 0.010000228881835938 ≤ 0.009999999776482582
+    » exact check is |9,989999771118164 - 10,0| = 0,010000228881835938 ≤ 0,009999999776482582
```

Comment:
please revert this change, was it intentional?

Reply:
It was edited automatically. I do not know why? Could be a language formatting.

Reply:
Ah... I see, makes sense, your local number formatting is different that's why and the example in the README gets automatically generated based on some specs.
Fancy to fix the problem so that someone else with a different number formatting does not run into the same problem?

Reply:
I do not know about automatically generated text in the README. I had a look a #211 and went tru the ReadmeSpec. I don't understand well how it is done. I would need to spend required time to dig it. Sadly, I am too busy right now. If I had the time, it would have been great for me to learn how to do it, like for this contribution.

Reply:
No worries, and you are always welcome to contribute again if you have time (also after hacktoberfest :grin: )